packages/spectrum/test/renderers/SpectrumAnyOfRenderer.test.tsx

```diff
@@ -215,30 +196,36 @@ describe('Spectrum anyOf renderer', () => {
       scope: '#/properties/myThingsAndOrYourThings',
     };
 
-    wrapper = mountForm(
+    const { container, getByRole } = renderForm(
       uischema,
       schema,
       {},
       undefined,
       ({ data }) => (state = data)
     );
+    await waitForAsync(); // TODO: how to do it with the testing-library...
+
+    let tablist = getByRole('tablist');
+    let tabs = within(tablist).getAllByRole('tab');
+    let firstItem = tabs[0];
+    let secondItem = tabs[1];
+    triggerPress(secondItem);
+
+    expect(secondItem).toHaveAttribute('aria-selected', 'true');
+
+    const addButton = container.querySelector('button');
+    userEvent.click(addButton);
 
     await waitForAsync();
 
-    wrapper.update();
+    // todo: there is no input field available... what is missing here?
```

Comment:
Is there something missing in the uischema? May we have to look at the original test...

Or we haven't implemented a component that is used here.